src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+
+Proof-of-stake data availability committees are considerably more secure than regular DACs. Not only are they permissionless and trustless, but they also have well-designed incentives to encourage honest behavior. 
+
+[More on proof-of-stake data availability committees.](https://blog.matter-labs.io/zkporter-a-breakthrough-in-l2-scaling-ed5e48842fbf) 
+
+### Data availability sampling {#data-availability-sampling}
+
+Data availability sampling is a cryptographic mechanism for guaranteeing data availability. Data availability sampling allows blockchain nodes to verify that data for a proposed block is available without having to download the entire block. 
+
+In a DAS system, a node samples small, random chunks of a block over multiple rounds to verify data availability. With many nodes sampling different parts of a block simultaneously, its  availability can be verified with high statistical certainty. 
+
+When applied to blockchains, like Ethereum, data availability sampling ensures light clients also participate in guaranteeing the chain's security and functionality. Light clients can run without expensive hardware, making it easier for anyone to validate on the Ethereum network. 
+
+[More on data availability sampling.](https://hackmd.io/@vbuterin/sharding_proposal#ELI5-data-availability-sampling) 
+
+### Data availability proofs {#data-availability-proofs}
+
+While DAS gives statistical guarantees of data availability, a malicious node can still hide some data. DAS techniques only prove that the majority of the block data is available, not that the entire block is available. And much harm can come from block producers hiding even a tiny amount of transaction data. 
+
+To solve this problem, we combine data availability sampling with [erasure coding](https://en.wikipedia.org/wiki/Erasure_code) to create “data availability proofs”. Erasure coding is a technique that allows us to double a dataset by adding redundant pieces (called erasure codes). If the original data is lost, the erasure codes can be used to reconstruct the original piece of data. 
+
+When implemented in blockchains, erasure codes improve data availability because a small fraction of the data is enough to reconstruct the whole transaction set in a block. In this system, a malicious block producer would need to withhold more than 50% of the block to perform a data withholding attack. Previously, a block producer only needed to seize 1% of block data to act maliciously. 
+
+With erasure-coded blocks, light clients have statistical certainty that the entire block data was published on the network. It also means that light clients don't have to rely on full nodes to alert them of the unavailability of a block. 
+
+[More on data availability proofs.](https://github.com/ethereum/research/wiki/A-note-on-data-availability-and-erasure-coding)
+
```

Comment:
This stops quite abruptly based on previous content. I'd expect some sort of wrap-up section here. Perhaps pointing to the solution, Ethereum is leaning toward or the further work being done? Not sure but it just feels a little blunt of a stop to me.

Reply:
Shouldn't we talk here about Danksharding?

Reply:
Could be nice - just to say something like "state-of-the-art scaling proposals such as Danksharding rely upon DAS to verify that the entire contents of a blob have been seen by the network, relieving individual nodes from the burden of downloading and validating it all directly". I think there is some nuance to whether this translates into data retrievability - afaik it doesn't.

Reply:
I already included some info about sharding and data availability in the "decentralized scalability" section, where I also talk about layer 2s. But, since you requested, I moved the sharding + DAS part to the concluding section and added @jmcook1186's addition about Danksharding.